typescript/.openapi-generator-ignore

```diff
@@ -1,22 +1,23 @@
 # OpenAPI Generator Ignore
-
-# Used by openapi-generator https://github.com/openapitools/openapi-generator
+# Generated by openapi-generator https://github.com/openapitools/openapi-generator
```

Comment:
why did this change? Is it from the new version? so maybe roll it back as well

Reply:
I do not know. I completely deleted the client folders, built the old generator image and regenerated the files.